DevEdu.API/DevEdu.DAL/Repositories/MaterialRepository.cs

```diff
@@ -0,0 +1,99 @@
+﻿using Dapper;
+using DevEdu.DAL.Models;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+
+namespace DevEdu.DAL.Repositories
+{
+    public class MaterialRepository : BaseRepository, IMaterialRepository
+    {
+        public MaterialRepository()
+        {
+
+        }
+
+        public int AddMaterial(MaterialDto material)
+        {
+            _query = "dbo.Material_Insert";
+            return _connection.QuerySingle<int>(
+                _query,
+                new { material.Content },
+                commandType: CommandType.StoredProcedure
+            );
+        }
+
+        public List<MaterialDto> GetAllMaterials()
+        {
+            _query = "dbo.Material_SelectAll";
```

Comment:
please create consts for all the stored procedures' names (as it was in the lesson)